### DISCUSSION TITLE
mts200 second temp entity

### DISCUSSION THREAD

**Discussion by schoppfe** [krahabb/meross_lan#509] [Q&A]
Can someone of you explain to me the second temperature entity?

After integration of my mts200 thermostat I noticed a second temperature entity. The first one updates regularly and seem to match the selected sensor of the device (air or ground) with an accuracy of 0.1 °C. The second sensor instead hasn’t changed for hours and seem to have an accuracy of only  0.5 °C. 

![IMG_4673](https://github.com/user-attachments/assets/a0a3778a-cefd-45b1-89d0-08d3dec78ffc)

from 17.11. 12:00 to 18.11. 00:00 I switched from air to ground sensor and then back.

**Comment by krahabb** [comment]
Might it be the 'external sensor temperature' ?
I guess meross_lan by default doesn't give it a very explicative name but maybe the entity_id could give some hints.
